templates/InstallChocolatey.ps1.erb

```diff
@@ -63,13 +64,21 @@ if ($unzipMethod -eq '7zip') {
 }
 
 # call chocolatey install
-Write-Host "Installing chocolatey on this machine"
+Write-Output "Installing chocolatey on this machine"
 $toolsFolder = Join-Path $tempDir "tools"
 $chocInstallPS1 = Join-Path $toolsFolder "chocolateyInstall.ps1"
 
-& $chocInstallPS1
+if ($PSVersionTable.psversion.Major -gt 2) {
+  & $chocInstallPS1
+}
+else {
+  $output = Invoke-Expression $chocInstallPS1
+  $output
+  write-output "Any errors that occured during install or upgrade are logged here: $chocoErrorLog"
+  $error | out-file $chocErrorLog
```

Comment:
i think this will help.  its an edge case thing, and ppl who encounter a real issue, we'll see why when they post their log, instead of the filestream bs.